src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/ApprenticeshipIncentiveQueryRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.EmployerIncentives.Abstractions.DTOs.Queries.ApprenticeshipIncentives;
+using SFA.DAS.EmployerIncentives.Data.Models;
+
+namespace SFA.DAS.EmployerIncentives.Data.ApprenticeshipIncentives
+{
+    public class ApprenticeshipIncentiveQueryRepository : IApprenticeshipIncentiveQueryRepository
+    {
+        private readonly EmployerIncentivesDbContext _context;
+
+        public ApprenticeshipIncentiveQueryRepository(EmployerIncentivesDbContext context)
+        {
+            _context = context;
+        }
+
+        public Task<List<ApprenticeshipIncentiveDto>> GetList()
+        {
+            return _context.Set<Models.ApprenticeshipIncentive>()
```

Comment:
Have seen a mix of usages of the DbSets on the EF context in the codebase. Is there a reason why you would use Set<T> here instead of _context.ApprenticeshipIncentives ?

Reply:
yes, I agree. I simply followed the other patterns set up for the query objects. I followed the pattern for the other Query repos. I'd assumed if we loaded _context.ApprenticeshipIncentives, it might load the related objects and hence do more than we need to. But I'm not 100% sure of what happens under the hood of EF

Reply:
Makes sense.